internal/keppel/security_scan_policy_test.go

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company
+// SPDX-License-Identifier: Apache-2.0
+
+package keppel
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/sapcc/keppel/internal/trivy"
+
+	"github.com/sapcc/go-bits/must"
+)
+
+func BenchmarkEnrichReport(b *testing.B) {
+	for b.Loop() {
+		policies := SecurityScanPolicySet{}
+		content := must.Return(os.ReadFile("../tasks/fixtures/trivy/report-vulnerable-with-fixes.json"))
+		report := trivy.ReportPayload{
+			Format:   "json",
+			Contents: io.NopCloser(bytes.NewReader(content)),
```

Comment:
I feel that it would be more faithful to the benchmark to use `os.Open()` directly here:
```suggestion
		report := trivy.ReportPayload{
			Format:   "json",
			Contents: must.Return(os.Open("../tasks/fixtures/trivy/report-vulnerable-with-fixes.json")),
```
And then `must.Succeed(file.Close())` at the end of the loop. (This should not use `defer` because then the benchmark may be impacted by open file handles piling up across loop iterations.)